lib/benchee/benchmark/scenario.ex

```diff
@@ -31,21 +31,21 @@ defmodule Benchee.Benchmark.Scenario do
   ]
 
   @type t :: %__MODULE__{
-    name:                    String.t,
-    job_name:                String.t,
-    function:                fun,
-    input_name:              String.t | nil,
-    input:                   any | nil,
-    run_times:               [float],
-    run_time_statistics:     Benchee.Statistics.t | nil,
-    memory_usages:           [non_neg_integer],
-    memory_usage_statistics: Benchee.Statistics.t | nil,
-    before_each:             fun | nil,
-    after_each:              fun | nil,
-    before_scenario:         fun | nil,
-    after_scenario:          fun | nil,
-    tag:                     String.t | nil
-  }
+          name: String.t(),
+          job_name: String.t(),
+          function: fun,
+          input_name: String.t() | nil,
+          input: any | nil,
+          run_times: [float],
+          run_time_statistics: Benchee.Statistics.t() | nil,
+          memory_usages: [non_neg_integer],
+          memory_usage_statistics: Benchee.Statistics.t() | nil,
+          before_each: fun | nil,
+          after_each: fun | nil,
+          before_scenario: fun | nil,
+          after_scenario: fun | nil,
+          tag: String.t() | nil
+        }
```

Comment:
this is odd?

Reply:
Yeah, I never really knew why the formatter aligns types this way, but that's how it works!

Reply:
seems inconsistent as they are essentially a hash? weird.